data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb

```diff
@@ -102,7 +102,9 @@
     "# Create the state ID by taking the first two digits of the FIPS CODE of the tract.\n",
```

Comment:
Just `black` running here. We should figure out why it's chaotically going back and forth.